ansible/extra/s390x.yml

```diff
@@ -0,0 +1,17 @@
+### S390X specifics (TODO: could I include what's above?) ###
+
+nginx_version: 1.12
```

Comment:
if doesn't hurt, let's just update the default from 1.11 to 1.12 and no need to override.
As curiosity version 1.12 has something required to work on s390x?

Reply:
1.12 docker image is available as multi-architecture image.  I can make the update in this PR, or in a separate PR after this is merged... your call.